<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminLoggedInScreen</class>
 <widget class="QDialog" name="AdminLoggedInScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>150</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>150</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>150</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Application/Database Management</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="lbl_heading">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Please Select an Option to Proceed</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radiobtnEmpMgnt">
       <property name="text">
        <string>Employee Management</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radiobtnDBMgnt">
       <property name="text">
        <string>Database Management</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radiobtnSkillMatrix">
       <property name="text">
        <string>Skill Matrix and Training</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbtnOK">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
